### PR TITLE
Use system hand cursor

### PR DESCRIPTION
**Pull request by sylveon**
Winforms had a hardcoded hand cursor because it originally supported Win98, which doesn't haves a system one. This change now uses the system cursor, because Windows 98 is obviously out of support.

### CODE REVIEW

**Comment by zsd4yr** [comment]
@sylveon would you mind filing a corresponding issue for this? Also I'm going to make an issue to replace these "magic" strings with const strings declared at the top of the file

**Comment by sylveon** [comment]
Sure! Should I fill this issue for all hardcoded cursors (since IIRC most can be replaced similarly) or just this one?

**Comment by merriemcgaw** [comment]
@sylveon , it would be lovely if everything could be consistent!  We may need to hold this PR until we've got adequate test infrastructure in place so you can write some tests. Hopefully we will also be able to test this offline when it's done and feedback incorporated, and then we'll let you know our results.  Thanks for the PR!

**Comment by sylveon** [comment]
There's no rush, take your time.

I've opened #140 for the changes and mentioned the need for unit tests.

File: src/System.Windows.Forms/src/System.Windows.Forms.csproj [hunk 1]
```diff
@@ -502,9 +502,6 @@
     <EmbeddedResource Include="Resources\System\Windows\Forms\east.cur">
       <LogicalName>System.Windows.Forms.east.cur</LogicalName>
     </EmbeddedResource>
-    <EmbeddedResource Include="Resources\System\Windows\Forms\hand.cur">
```
Comment (daniel-white) [inline src/System.Windows.Forms/src/System.Windows.Forms.csproj]: should the file be deleted as well?

**Comment by zsd4yr** [comment]
@sylveon just wondering -- do you plan to take on issue #140 as part of this PR (or as separate or not at all)? If not, would you like us to go ahead and approve / merge this once we're comfortable ?

**Comment by sylveon** [comment]
@zsd4yr I'm planning on writing the tests this weekend and depending how it goes I might do it.

**Comment by zsd4yr** [comment]
Sounds great :) let me know if you need anything from us and I’ll look first thing Monday

**Comment by sylveon** [comment]
I've pushed some basic testing for the Cursors class. Unfortunately I didn't manage to replace the other hardcoded cursors because they have no direct system equivalent.

**Comment by zsd4yr** [comment]
I will rereview this soon.

**Pull request merged**